packages/edit-site/src/components/left-sidebar/navigation-panel/template-switcher/index.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+import {
+	Tooltip,
+	__experimentalNavigationGroup as NavigationGroup,
+	__experimentalNavigationItem as NavigationItem,
+} from '@wordpress/components';
+import { Icon, home, plus, undo } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import TemplatePreview from './template-preview';
+import ThemePreview from './theme-preview';
+
+const TEMPLATE_OVERRIDES = {
+	page: ( slug ) => `page-${ slug }`,
+	category: ( slug ) => `category-${ slug }`,
+	post: ( slug ) => `single-post-${ slug }`,
+};
+
+function TemplateLabel( { template, homeId } ) {
+	return (
+		<>
+			{ template.slug }{ ' ' }
+			{ template.id === homeId && (
+				<Tooltip text={ __( 'Home' ) }>
+					<div className="edit-site-template-switcher__label-home-icon">
+						<Icon icon={ home } />
+					</div>
+				</Tooltip>
+			) }
+			{ template.status !== 'auto-draft' && (
+				<Tooltip text={ __( 'Customized' ) }>
+					<span className="edit-site-template-switcher__label-customized-dot" />
+				</Tooltip>
+			) }
+		</>
+	);
+}
+
+function NavigationItemWithIcon( { icon, title, ...props } ) {
+	return (
+		<NavigationItem
+			title={
+				<>
+					{ title }{ ' ' }
+					{ icon && (
+						<div className="edit-site-template-switcher__label-home-icon">
+							<Icon icon={ icon } />
+						</div>
```

Comment:
`TemplateLabel` wrapped the icon in `Tooltip`.
I think it's a nice and easy a11y win if we brought the same feature here.
We could add an `iconLabel` prop for the tooltip, that would default to `title` if not provided.

Reply:
Done!